src/main/java/io/leonis/zosma/ipc/peripheral/component/Bumper.java

```diff
@@ -0,0 +1,30 @@
+package io.leonis.zosma.ipc.peripheral.component;
+
+/**
+ * The class Bumper.
+ *
+ * Describes the buttons typically on the top of a controller, such as the shoulder buttons on a
+ * XBOX Controller.
+ *
+ * @author Jeroen de Jong
+ */
+public final class Bumper {
```

Comment:
i don't like having this as a class

Reply:
What do yo propose?

Reply:
either turn this into a (useless) interface or extract the interfaces